build/platform_and_feature_flags.props

```diff
@@ -26,4 +26,9 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD;SUPPORTS_CONFIDENTIAL_CLIENT;SUPPORTS_BROKER;SUPPORTS_CUSTOM_CACHE;SUPPORTS_OS_SYSTEM_BROWSER;SUPPORTS_WEBVIEW2</DefineConstants>
   </PropertyGroup>
+
+  <PropertyGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
```

Comment:
NOTE: The AAD Kerberos Feature is targeting only the Windows platform now. We added the SUPPORT_KERBEROS flag to count on this limitation.

Reply:
Since this is ultimatly a windows target, maybe we can simply call it "WINDOWS". Similar to what we have for "iOS", "MAC" and "ANDROID". Then we can use it for other features that may come in the future.